buildlib/tools/common.sh

```diff
@@ -34,3 +36,22 @@ make_clean() {
 	rm -rf ${ucx_inst}
 	$MAKEP ${1:-clean}
 }
+
+#
+# Prepare build environment
+#
+prepare() {
+	echo " ==== Prepare ===="
+	az_init_modules
+	env
+	cd ${WORKSPACE}
+	if [ -d ${ucx_build_dir} ]
+	then
+		chmod u+rwx ${ucx_build_dir} -R
+		rm -rf ${ucx_build_dir}
+	fi
+	./autogen.sh
+	mkdir -p ${ucx_build_dir}
+	cd ${ucx_build_dir}
+	export PROGRESS=0
```

Comment:
what is this?

Reply:
for azure view, added progress bar for build process 

```sh
do_task() {
	$@
	echo "##vso[task.setprogress value=$PROGRESS;]Progress Indicator"
	PROGRESS=$((PROGRESS+5))
}
```

Reply:
hah nice